### PR TITLE
feat: add tutor plugin

### PR DESCRIPTION
**Pull request by Henrrypg**
# Description

This PR takes drydock-backups from [drydock](https://github.com/eduNEXT/drydock/tree/main/drydock_backups).

### CODE REVIEW

**Comment by Ian2012** [comment]
@Henrrypg any github workflow we should implement here?

**Comment by Henrrypg** [comment]
@Ian2012 I added release and commitlint workflows for now, if you are thinking in something else let me know

**Review by Ian2012 — APPROVED** [review]

**Review by marbonilla — APPROVED** [review]

File: drydock_backups/plugin.py [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+
+import os
+import os.path
+from glob import glob
+
+import click
+import pkg_resources
+from tutor import hooks
+
+from .__about__ import __version__
```
Comment (MoisesGSalas) [inline drydock_backups/plugin.py]: Do we start again from version 0.1.0?

**Review by MoisesGSalas — COMMENTED** [review]
Does it make sense to restore the history similar to how we split the shipyard mono repo?

**Pull request closed without merging**